### PR TITLE
chore: add support for Python 3.13

### PR DESCRIPTION
**Pull request by timoo4devv**
Support Python 3.13

Depends on #2176

### CODE REVIEW

**Comment by Paillat-dev** [comment]
Does the lib actually work in `3.13`? I believe some issues with voice need to be fixed first.

Either  #2176 is merged or `audioop-lts~=0.2.1` is added to req.

**Review by Lulalaby — APPROVED** [review]

**Comment by Paillat-dev** [comment]
Pr should also update `README.rst` here:

https://github.com/Pycord-Development/pycord/blob/0f1a079836ccad8d7e1db917f9066414220def79/README.rst#L34

File: .readthedocs.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ formats: []
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.12"
+    python: "3.13"
```
Comment (Lulalaby) [inline .readthedocs.yml:7]: Please do not touch readthedocs
This version is untested

**Review by Lulalaby — CHANGES_REQUESTED** [review]

**Comment by Lulalaby** [comment]
We're gonna handle the rest internally for now.
There are still dependency issues so we can't merge it into main

**Pull request merged**